content_handler/runtime_holder.cc

```diff
@@ -302,7 +302,7 @@ std::string RuntimeHolder::DefaultRouteName() {
   return "/";
 }
 
-void RuntimeHolder::ScheduleFrame() {
+void RuntimeHolder::ScheduleFrame(bool regenerate_layer_tree) {
```

Comment:
Perhaps make a comment noting that regenerate_layer_tree is ignored

Reply:
Done.